src/query/procedure/mg_procedure_impl.cpp

```diff
@@ -1052,6 +1115,11 @@ mgp_error mgp_map_copy(mgp_map *map, mgp_memory *memory, mgp_map **result) {
 
 void mgp_map_destroy(mgp_map *map) { DeleteRawMgpObject(map); }
 
+mgp_error mgp_map_contains_deleted(mgp_map *map, int *result) {
+  *result = ContainsDeleted(map);
```

Comment:
I think we should use WrapExceptions, don't know what might happen here

Reply:
I didn’t envision any case where this function would throw an exception, but I can add it no problem